timeutils.py

```diff
@@ -1,37 +1,39 @@
 import logging
-from datetime import timedelta, timezone, datetime
+from datetime import timedelta, timezone
+
 logger = logging.getLogger(__name__)
 
 MEETUP_TZ = timezone(timedelta(hours=-4))
 HS_TZ = timezone(timedelta(hours=2))
 
+
 def meetup_time(dt):
     """Converts time to format accepted by meetup API"""
     logger.debug("%s, %s", dt, dt.timestamp())
     logger.info('change timezone')
     dt = dt.replace(tzinfo=HS_TZ)
     logger.debug("%s, %s", dt, dt.timestamp())
-    result = datetime2ms(dt)
```

Comment:
I don't think it's uncommon to see such names

Reply:
* http://pep8.org/#descriptive-naming-styles